src/mudlet.qrc

```diff
@@ -72,6 +72,7 @@
         <file>icons/face-smile.png</file>
         <file>icons/face-surprise.png</file>
         <file>icons/face-uncertain.png</file>
+        <file>icons/fiery_mud.png</file>
```

Comment:
👍  For inserting it in the (non-case sensitive) sorted position instead of at the bottom - I try to keep this file sorted so it is easier to find each resource...